### PR TITLE
fix: Remove potential FPE in `RootTrajectorySummaryWriter`

### PR DESCRIPTION
**Pull request by andiwand**
`NAN` default values are evil

### CODE REVIEW

File: Examples/Io/Root/src/RootTrajectorySummaryWriter.cpp [hunk 1]
```diff
@@ -345,22 +347,21 @@ ActsExamples::ProcessCode ActsExamples::RootTrajectorySummaryWriter::writeT(
...
+              param[Acts::eBoundQOverP] - t_qop,
+              param[Acts::eBoundTime] - t_time};
+
+          if (boundParam.covariance().has_value()) {
+            const auto& covariance = *boundParam.covariance();
+            for (unsigned int i = 0; i < Acts::eBoundSize; ++i) {
+              error[i] = std::sqrt(covariance(i, i));
               pull[i] = res[i] / error[i];
-            } else {
-              pull[i] = std::numeric_limits<double>::quiet_NaN();
```
Comment (paulgessinger) [inline Examples/Io/Root/src/RootTrajectorySummaryWriter.cpp:363]: But then the vector sizes are not the same anymore, are they?

Comment (andiwand) [reply Examples/Io/Root/src/RootTrajectorySummaryWriter.cpp:363]: I think it should be. my goal is to write `NAN` if we don't know the reconstructed params or true params but maybe I messed this up

Comment (paulgessinger) [reply Examples/Io/Root/src/RootTrajectorySummaryWriter.cpp:363]: Ok i might have to reread the diff more closely.

**Comment by andiwand** [comment]
I do not expect any hash changes. Lets see if this is fixed now

**Review by paulgessinger — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results
### Build job with this PR failed!
Please investigate the [build job](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30813174) for the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/5845949)!

**Pull request merged**